hw2/03-count.html

```diff
@@ -9,25 +9,27 @@
       href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
     <script defer src="03-count.js"></script>
   </head>
-  <body>
+  <body style="background-color: #01435C;">
```

Comment:
In-line CSS is considered bad practice because of it's difficulty to maintain. Instead just add a class to the element and define the styles in a separate CSS file.